<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Graphing Thing!</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPushButton" name="button1">
     <property name="text">
      <string>START</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="button2">
     <property name="text">
      <string>STOP</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="button3">
     <property name="text">
      <string>Done</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="GraphWidget" name="plot" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GraphWidget</class>
   <extends>QWidget</extends>
   <header>graph_widget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>button1</sender>
   <signal>clicked()</signal>
   <receiver>plot</receiver>
   <slot>repaint()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>399</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>350</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
